tox.ini

```diff
@@ -1,20 +1,20 @@
 [tox]
-envlist = py{26,27,py,33,34,35}-{test,uwsgi,stylecheck}
+envlist = py{26,27,py,33,34,35}-{normal,uwsgi,stylecheck}
 
 [testenv]
 passenv = LANG
 deps=
 # General
-    pyopenssl
-    greenlet
-    pytest
-    pytest-xprocess
-    redis
-    requests
-    watchdog
-
-    uwsgi: uwsgi
+    normal: pyopenssl
```

Comment:
I just want to spare the redundant `normal: `-prefixes. I'd like to do something like this:

    [testenv:normal]
    deps=
        pyopenssl
        ....

Unfortunately this doesn't work.